certbot-nginx/tests/obj_test.py

```diff
@@ -168,7 +168,7 @@ def test_eq(self):
 
         self.assertEqual(vhost1b, self.vhost1)
         self.assertEqual(str(vhost1b), str(self.vhost1))
-        self.assertFalse(vhost1b == 1234)
+        self.assertEqual(vhost1b, 1234)
```

Comment:
This should be `assertNotEqual`, what is going on ...

Reply:
Hmm. I've updated it. Hopefully this emphasizes my point that those asserts can be more difficult to read ...

Reply:
To be clear, the "what is going on" was about the fact that the tests were "passing", until I understood that they were not run. I completely agree with you that several assertions were double negation, so quite difficult to read properly.